insights/parsers/cpupower_frequency_info.py

```diff
@@ -0,0 +1,88 @@
+"""
+CpupowerFrequencyInfo - Commands ``cpupower frequency-info``
+============================================================
+"""
+from insights import parser, CommandParser, LegacyItemAccess
+from insights.parsers import SkipException, ParseException
+from insights.specs import Specs
+
+
+@parser(Specs.cpupower_frequency_info)
+class CpupowerFrequencyInfo(CommandParser, LegacyItemAccess):
```

Comment:
The `LegacyItemAccess` is not suggested to be used to access the `self.data`, please mix it with `dict` directly,

For your reference: 

https://github.com/RedHatInsights/insights-core/blob/532ad3d8c6cebec9dff0e49a151307d4ab7713a0/insights/parsers/modinfo.py#L145

https://github.com/RedHatInsights/insights-core/blob/532ad3d8c6cebec9dff0e49a151307d4ab7713a0/insights/parsers/modinfo.py#L226